api/billingd/service/service.go

```diff
@@ -1004,8 +1019,12 @@ func (s *Service) reportCustomerUsage(ctx context.Context, cus *Customer) error
 	return nil
 }
 
-func (s *Service) reportUnits(product Product, usage Usage) error {
+func (s *Service) reportUnits(product Product, usage Usage, parentKey string) error {
 	freeUnits, paidUnits := getUnits(product, usage.Total)
+	if parentKey != "" {
+		freeUnits += paidUnits
+		paidUnits = 0
+	}
```

Comment:
I meant to add this already. If this is a dependent account, report all units to the free subscription item. This may have caused an issue in production where some heavy users have had daily usage reported to the paid subscription item, but they're still protected by the grace period. We can do some searching in Stripe to find any affected users.